lib/resque/scheduler.rb

```diff
@@ -208,18 +210,18 @@ def enqueue_delayed_items_for_timestamp(timestamp)
         end
       end
 
+      def enqueue(config)
+        enqueue_from_config(config)
+      rescue => e
+        Resque::Scheduler::Plugin.run_failure_hooks(failure_handler, config, e)
```

Comment:
We should keep the default logging behaviour if the backend isn't specified.  I'm not sure if that should be inside the failure handler or not.